odk1-src/form-functions.rst

```diff
@@ -0,0 +1,719 @@
+.. spelling::
+
+  Anaranjado
+  Azul
+  Colores
+  Español
+  Púrpura
+  Qué
+  Rojo
+  Seleccione
+  arctan
+  arg
+  bday
+  bp
+  br
+  chkn
+  colores
+  fsh
+  gustan
+  jr
+  nodeset
+  prefs
+  seleccionados
+  te
+  tres
+  π
+  
+  
+	
+******************************
+Form Functions and Operators
+******************************
+
+:ref:`expressions` in :ref:`calculations <calculations>`, :ref:`constraints <constraints>`, and :ref:`relevants <relevants>`
+can contain operators and functions.
+
+.. seealso:: `XPath Functions <https://opendatakit.github.io/xforms-spec/#xpath-functions>`_
+
+.. contents::
+  :local:   
+
+  
+.. _xform-operators:
+
+Operators
+==========
+
+.. _utility-operators:
+
+Utility operators
+-------------------
+
+.. csv-table::
+  :header: , Explanation, Example, Notes
+  
+  ., current question's value, . >= 18, Used in :ref:`constraints <constraints>`.
+  \.\., current question's parent group, position(..), Used with :func:`position` to get the iteration index.
+  
+.. _boolean-operators:
+  
+Boolean operators
+------------------
+
+.. csv-table::
+  :header: , Explanation, Example
+
+  and, ``True`` if the expressions before and after are ``True``, ${age} > -1 and ${age} < 120
+  or, ``True`` if either of the expressions before or after are ``True``, ${age} < 19 or ${age} > 64
+  
+.. _comparison-operators:
```

Comment:
I would tend to put comparison operators before boolean operators. Most people aren't familiar with the word "boolean". Perhaps there can be a short sentence when introducing comparison operators that says that the result is always `true()` or `false()` (boolean). Then the boolean operators introduction can say they join two boolean expressions (that evaluate to `true()` or `false()`).